.github/workflows-source/ci-pr-only.yml

```diff
@@ -3,9 +3,11 @@ name: CI PR Only
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened]
+    types: [opened, edited, synchronize, reopened]
```

Comment:
My PR will have all the workflows green, I decide to update the title,  that will effectively trigger `lock-generate`, `dependencies-check` and `pre-commit`. Same goes for updating description of the PR. This can stay how it is.

Reply:
Hmm okay I'll keep the workflow and rename it to something a little clearer then.